internal/xds/translator/authentication.go

```diff
@@ -195,34 +195,25 @@ func buildClusterFromJwks(jwks *jwksCluster) (*cluster.Cluster, error) {
 		return nil, errors.New("jwks is nil")
 	}
 
-	endpoints, err := jwks.toLbEndpoints()
+	destinations, err := jwks.toRouteDestinations()
 	if err != nil {
 		return nil, err
 	}
 
+	cluster := buildXdsCluster(jwks.name, destinations, false /*isHTTP2 */)
+	cluster.ConnectTimeout = durationpb.New(10 * time.Second)
+	cluster.DnsRefreshRate = durationpb.New(30 * time.Second)
```

Comment:
I believe this field can be removed since the cluster type is static or you can try moving the JWKS cluster type to DNS and remove the code that converts the JWKS URL hostname to IP addresses.

Reply:
thinking about this again, if the API supports an IP addr as a Hostname then maybe this layer can decide based on the str and create a static or dynamic cluster, dont think this needs to be part of this PR since its out of scope from the issue this PR is fixing

Reply:
changed JWKS cluster type to STRICT_DNS

Reply:
I dont think STRICT_DNS is right for this case (sorry if I said the opposite before)
from the API spec https://gateway.envoyproxy.io/v0.3.0/api/extension_types.html#remotejwks
the hostname section within the URI could be an IP Addr or a FQDN. 
https://github.com/envoyproxy/gateway/blob/17d325d3ff59e04a045fff49a1edaafe3f633314/internal/xds/translator/authentication.go#L480
current converts hostname into IP string, so this cluster must be STATIC

In the future this code case could be enhanced to eliminate resolveHostname and decide discovery type based on Hostname type